src/main/java/com/adscore/signature/SignatureVerifier.java

```diff
@@ -32,7 +32,11 @@
  */
 public class SignatureVerifier {
 
+  public static final int DEFAULT_EXPIRY_TIME_SEC = 60;
```

Comment:
this probably should be noted in readme.

Reply:
Good point. Added comments to examples that don't have expiry param (so the default is used).